<launch>

  <arg name="move_base_type" default="static"/>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">

    <rosparam file="$(find magni_octonav)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find magni_octonav)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />

    <rosparam file="$(find magni_octonav)/config/$(arg move_base_type)/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find magni_octonav)/config/$(arg move_base_type)/global_costmap_params.yaml" command="load" />

    <rosparam file="$(find magni_octonav)/config/base_local_planner_params.yaml" command="load" />
    <rosparam file="$(find magni_octonav)/config/base_global_planner_params.yaml" command="load" />
    <rosparam file="$(find magni_octonav)/config/move_base_params.yaml" command="load" />

    <param name="base_global_planner" value="global_planner/GlobalPlanner" />
    <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>

    <!-- <remap from="odom" to="/odometry/filtered" /> -->
    <!-- <remap from="odom" to="/ubiquity_velocity_controller/odom" /> -->
    <remap from="cmd_vel" to="/cmd_vel/nav" />
  </node>

</launch>
